lib/app/service/nekoton_related/connection_service/transport_strategies/custom_transport_strategy.dart

```diff
@@ -69,6 +77,8 @@ class CustomTransportStrategy extends TransportStrategy {
 
   @override
   String transactionExplorerLink(String transactionHash) => '';
+  // TODO(nesquikm): get transaction explorer link from connection data
+  //    '${connection.transactionExplorerURL}/$transactionHash';
```

Comment:
For transaction and account explorer must be same url with different paths, see `EverTransportStrategy` (imho)

Reply:
And where should we get the base url from for the custom network?

Reply:
In extension, this is `Block explorer link`

Reply:
Ah, okay. Will fix this info in the google doc.